sechub-commons-model/src/test/java/com/mercedesbenz/sechub/commons/model/JSONConverterTest.java

```diff
@@ -19,6 +24,209 @@ void before() {
+
+        /* execute */
+        String json = converterToTest.toJSON(origin);
+
+        /* test */
+        assertTrue(json.contains("["));
+    }
+
+    @Test
+    void toJson_string_collection_with_one_element_is_an_array() {
+        /* prepare */
+        CollectionTestClass origin = new CollectionTestClass();
+        origin.collection.add("alpha");
+
+        /* execute */
+        String json = converterToTest.toJSON(origin);
+
+        /* test */
+        assertTrue(json.contains("["));
+    }
+
+    @Test
+    void toJson_enum_is_uppercased() {
+        /* prepare */
+        TrafficLightEnumTestClass origin = new TrafficLightEnumTestClass();
+        origin.trafficLight = TrafficLight.GREEN;
+
+        /* execute */
+        String json = converterToTest.toJSON(origin);
+
+        /* test */
+        assertNotNull(json);
+        assertTrue(json.contains("GREEN"));
+        assertFalse(json.contains("green"));
+
+    }
+
+    @Test
+    void fromJson_enum_as_lowercased_can_be_read() {
+        /* prepare */
+        TrafficLightEnumTestClass origin = new TrafficLightEnumTestClass();
+        origin.trafficLight = TrafficLight.GREEN;
+
+        String json = converterToTest.toJSON(origin);
+        String jsonWithLoweredGreen = json.replaceAll("GREEN", "green");
+
+        assertNotNull(json);
+        assertTrue(jsonWithLoweredGreen.contains("green"));
+        assertFalse(jsonWithLoweredGreen.contains("GREEN"));
+
+        /* execute */
+        TrafficLightEnumTestClass result = converterToTest.fromJSON(TrafficLightEnumTestClass.class, jsonWithLoweredGreen);
+
+        /* test */
+        assertEquals(TrafficLight.GREEN, result.trafficLight);
+
+    }
+
+    @Test
+    void toJson_for_a_object_containing_a_local_date_must_work() {
+        /* prepare */
+        LocalDateTestClass testObject = new LocalDateTestClass();
+        testObject.date = LocalDate.now();
+
+        /* execute */
+        String json = converterToTest.toJSON(testObject);
+
+        /* test */
+        assertNotNull(json);
+
+    }
+
+    @Test
+    void toJson_and_from_json_local_date_must_be_equal() {
+        /* prepare */
+        LocalDateTestClass origin = new LocalDateTestClass();
+        origin.date = LocalDate.now();
+
+        /* execute */
+        String json = converterToTest.toJSON(origin);
+        System.out.println(json);
+
+        /* test */
+        assertNotNull(json);
+
+        /* execute */
+        LocalDateTestClass result = converterToTest.fromJSON(LocalDateTestClass.class, json);
+        assertEquals(origin.date, result.date);
+
+    }
+
+    @Test
+    void fromJson_with_array_localdatetime_works() {
+        /* prepare */
+        String json = "{\"dateTime\":[2023,3,7,10,38,49,470191000]}";
+
+        /* execute */
+        LocalDateTimeTestClass result = converterToTest.fromJSON(LocalDateTimeTestClass.class, json);
+        assertNotNull(result.dateTime);
+        assertEquals(2023, result.dateTime.getYear());
```

Comment:
Please add a second assert equals which tests the date time component. For example the seconds or nano-seconds.